api/report_server.thrift

```diff
@@ -158,6 +183,12 @@ struct CommentData {
 }
 typedef list<CommentData> CommentDataList
 
+
+struct CompareData {
```

Comment:
Let's introduce a comment here, how this method works, and how passing a `CompareData` to features that do take a `CompareData` argument changes the behaviour.